proto/net.proto

```diff
@@ -106,7 +106,7 @@ message NetworkFormat {
   enum OutputFormat {
     OUTPUT_UNKNOWN = 0;
     OUTPUT_CLASSICAL = 1;
-    // OUTPUT_WDL = 2;  // Example. Uncomment when implemented.
+    OUTPUT_WDL = 2;
```

Comment:
The same argument crem is making for the A0 policy head to add a new enum to network structure would seem to also apply to this PR, as unfortunately I don't see any code that validates that OutputFormat is classical in the current lc0 codebase.

Reply:
I didn't look how this enum is going to be used, but it was intended to be used by search code only.
So if all WDL-related stuff happens inside backends and search code doesn't have to know about it, this enum doesn't have to be touched.
But I guess we'll have to propagate draw probability out of backend to nodes via some change of NetworkComputation interface? If yes, then adding value into enum is what is needed.

Reply:
Search code is propagating D values so this change makes sense - just a question of whether it is required to have a change to the network format as well.